### PR TITLE
cylc gcylc/graph: hide suicide triggers as default

### PR DESCRIPTION
**Pull request by kaday**
Hides the suicide triggers with cylc gcylc and cylc graph.
Option --show-graph for cylc graph so that it opens with the graph displaying.

### CODE REVIEW

**Comment by kaday** [comment]
@matthewrmshin please review 1
@arjclark please review 2

**Comment by hjoliver** [comment]
@kaday - why would we not want to show the graph at start-up?   Actually your new `--show-graph` option only affects whether or not suicide triggers are shown at start-up, so maybe you meant it to be `--show-suicide`?  (In which case the logic could simply set the value of the `ignore_suicide` boolean?)

**Comment by matthewrmshin** [comment]
@kaday Could you add a test to make use of the `--show-graph` (or `--show-suicide` as suggested by @hjoliver) option? (A copy of the modified test should be sufficient, and should probably behave like it did before this change.)

**Comment by kaday** [comment]
@hjoliver yes you are correct the name was wrong. I have changed this to your suggestion.
@matthewrmshin test added.

**Comment by matthewrmshin** [comment]
Now looking OK 1. Tests pass.

**Comment by kaday** [comment]
@hjoliver please review
@arjclark give this one a miss

File: bin/cylc-graph [hunk 1]
```diff
@@ -178,6 +178,12 @@ def main():
              "If not given, assume --output-file=-.",
         action="store_true", default=False, dest="reference")
 
+    parser.add_option(
+        "--show-suicide",
+        help="Show the suicide triggers, default is to not display the graph "
+             "and with the suicide triggers.",
```
Comment (hjoliver) [inline bin/cylc-graph]: Is the 'and' here a typo?  Maybe something like this:
```
Show suicide triggers.  They are not shown by default, unless toggled on with the tool bar button.
```

File: doc/cug.tex [hunk 2]
```diff
@@ -3447,7 +3447,9 @@ \subsubsection{Trigger Types}
 suite (you can run the {\em AutoRecover.async} example suite to see how
 this works).  The dashed graph edges ending in solid dots indicate
 suicide triggers, and the open arrowheads indicate conditional triggers
-as usual.
+as usual. Note as default the suicide triggers are ignored in the graphing. 
+Under {\em View}, {\em Options} you can select to toggle 
+{\em Ignore Suicide Triggers}.
```
Comment (hjoliver) [inline doc/cug.tex]: I'm being pedantic sorry, but I "as default" and "ignored in the graphing" don't sound right to me.  I'd prefer something like *Suicide triggers are ignored by default in the graph view, unless you toggle them on with View -> Options -> Ignore Suicide Triggers*.

**Comment by hjoliver** [comment]
Looks good now, just two comments on the text.

**Comment by kaday** [comment]
@hjoliver your suggestions have been applied

**Comment by hjoliver** [comment]
@kaday - good, thanks.

**Pull request merged**